src/Tizen.NUI/src/public/Window/DefaultBorder.cs

```diff
@@ -824,7 +825,8 @@ private void UpdateIcons()
                     {
                         rightCornerIcon.ResourceUrl = DarkRightCornerIcon;
                     }
-                    borderView.CornerRadius = new Vector4(0, 0, 0, 0);
+                    using var tmp = ObjectPool.NewVector4(0, 0, 0, 0);
+                    borderView.CornerRadius = tmp;
```

Comment:
```suggestion
                    borderView.CornerRadius = Vector4.Zero;
```

Minor. Couldn't we use `Vector4.Zero` value directly for those zero cases?